features/product/managing_product_association_types/filtering_product_association_types.feature

```diff
@@ -0,0 +1,24 @@
+@managing_product_association_types
+Feature: Filtering product association types
+    In order to filter product association types by name or code
```

Comment:
Do we have to be such precise? If new feature would be added should be change also description?
`In order to filter product association types`